composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.0.20",
+        "composer/installers": "dev-master",
```

Comment:
Could we append the current commit revision here, so it isn't always pulling `dev-master`?

`composer/installers: dev-master#f00o0oo0oo0oo` for example